graaljs/graaljs-spring-boot-d3/src/main/resources/static/graph.js

```diff
@@ -0,0 +1,85 @@
+var d3 = require('./node_modules/d3/dist/d3.js');
+var linkedom = require('linkedom');
+
+globalThis.document = linkedom.parseHTML('<html><body></body></html>').document;
+
+const width = 600;
+const height = 600;
+const margin = 50;
+const innerRadius = Math.min(width, height) * 0.5 - 60;
+const outerRadius = innerRadius + 20;
+
+const data = [
+    [0, 20, 10, 5, 8, 3, 12, 1],
+    [20, 0, 15, 7, 6, 9, 4, 2],
+    [10, 15, 0, 12, 3, 5, 6, 8],
+    [5, 7, 12, 0, 10, 1, 7, 3],
+    [8, 6, 3, 10, 0, 14, 9, 11],
+    [3, 9, 5, 1, 14, 0, 8, 6],
+    [12, 4, 6, 7, 9, 8, 0, 10],
+    [1, 2, 8, 3, 11, 6, 10, 0]
+];
+
+const names = ["Java", "JavaScript", "Python", "Ruby", "R", "C/C++", "Native Image", "Polyglot"];
```

Comment:
I like that you tried to come up with some graph related to GraalVM. However, this chord diagram between different languages and toolchains does not make a lot of sense to me.

It would be absolutely fine, and maybe even a good idea, to use an off-the-shelf example from D3, like this one: https://observablehq.com/@d3/chord-diagram/2

Just add a comment that references the original example.